pephub/routers/eido.py

```diff
@@ -139,13 +280,18 @@ async def validate_pep(
     # return HTMLResponse(je.get_template("validation_results.html").render(**vals))
 
 
+@router.get("/")
+async def main():
+    print(je.list_templates())
+    return HTMLResponse(je.get_template("hello.html").render())
+
+
 # @router.get("/")
 # async def main():
-#     print(je.list_templates())
-#     return HTMLResponse(je.get_template("index.html").render())
+#     return HTMLResponse(je.get_template("schema.html").render())
 
 
 @router.get("/validator")
 async def main():
     print(je.list_templates())
-    return FileResponse(os.path.join(STATICS_PATH, "index.html"))
+    return FileResponse(os.path.join(STATICS_PATH, "index2.html"))
```

Comment:
Can we delete `index.html` and rename `index2.html` to `index.html`? Does that make sense?